doc/10-icinga-template-library.md

```diff
@@ -736,6 +736,66 @@ http_link                        | **Optional.** Wrap output in HTML link. Defau
 http_verbose                     | **Optional.** Show details for command-line debugging. Defaults to false.
 http_verify_host                 | **Optional.** Verify SSL certificate is for the -H hostname (with --sni and -S). Defaults to false. **Only supported by the Nagios plugins version of check\_http, not by the monitoring plugins one.**
 
+### curl <a id="plugin-check-command-curl"></a>
+
+The [check_curl](https://www.monitoring-plugins.org/doc/man/check_curl.html) plugin
+tests the HTTP service on the specified host. It can test normal (http) and secure
+(https) servers, follow redirects, search for strings and regular expressions,
+check connection times, and report on certificate expiration times.
+
+The plugin can either test the HTTP response of a server, or if `curl_certificate_valid_days_min_warning`/`curl_certificate_valid_days_min_critical` is set to a non-empty value, the TLS certificate age for a HTTPS host.
+
+Custom variables passed as [command parameters](03-monitoring-basics.md#command-passing-parameters):
+
+Name                             | Description
+---------------------------------|---------------------------------
+curl_vhost                       | **Optional.** The virtual host that should be sent in the "Host" header.
+curl_ip							 | **Optional.** The host's address. Defaults to "$address$" if the host's `address` attribute is set, "$address6$" otherwise.
+curl_port                        | **Optional.** The TCP port. Defaults to 80 when not using SSL, 443 otherwise.
+curl_ipv4                        | **Optional.** Use IPv4 connection. Defaults to false.
+curl_ipv6                        | **Optional.** Use IPv6 connection. Defaults to false.
+curl_tls						 | **Optional.** Whether to use SSL. Defaults to false.
+curl_tls_version	             | **Optional.** Connect via SSL. Port defaults to 443. VERSION is optional, and prevents auto-negotiation (2 = SSLv2, 3 = SSLv3, 1 = TLSv1, 1.1 = TLSv1.1, 1.2 = TLSv1.2, 1.3 = TLSv1.3). With a '+' suffix, newer versions are also accepted. Note: SSLv2 and SSLv3 are deprecated and are usually disabled in libcurl.
+curl_sni                         | **Optional.** Whether to use SNI. Defaults to false.
```

Comment:
The man page states:
```
 --sni
    Enable SSL/TLS hostname extension support (SNI)
    Note: --sni is the default in libcurl as SSLv2 and SSLV3 are deprecated and
          SNI only really works since TLSv1.0
```

Your short version of the text reads like SNI would be disabled. However, in the current age, SNI is the default.

Reply:
edited the description slightly to make this more visible